File: app/Android/Expense.kt
package com.example


/**
* Homeflow Technologies | Expense.
*
* @property id
* @property user_id
* @property category_id
* @property amount
*
* @constructor Create Expense model
*/
data class Expense( var user_id: Int? = null, var category_id: Int? = null, var amount: Double? = null,
)

File: app/Models/Expense.php
<?php

namespace App\Models;

use App\Models\Scopes\Searchable;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Expense extends Model
{
    use HasFactory;
    use Searchable;

    protected $fillable = ['user_id', 'category_id', 'amount'];

    protected $searchableFields = ['*'];

    public function user()
    {
        return $this->belongsTo(User::class);
    }

    public function category()
    {
        return $this->belongsTo(Category::class);
    }
}
